apps/web/app/routes/app/proposals/_index/_route.tsx

```diff
@@ -38,7 +39,14 @@ export default function Index() {
             <>
               <Card className="pb-10">
                 <CardHeader>
-                  <CardTitle>Active Standard Proposals</CardTitle>
+                  <div className="flex items-center justify-between">
+                    <CardTitle>Active Standard Proposals</CardTitle>
+                    <a href={$path("/app/proposals/new")}>
+                      <Button data-testid="start-regular-upgrade" variant="secondary" size="icon">
+                        <PlusIcon className="h-4 w-4" />
+                      </Button>
+                    </a>
```

Comment:
Using the Remix `Link` component allows for app navigation instead of full browser navigation.

```suggestion
                    <Link href={$path("/app/proposals/new")}>
                      <Button data-testid="start-regular-upgrade" variant="secondary" size="icon">
                        <PlusIcon className="h-4 w-4" />
                      </Button>
                    </Link>
```